packages/framework-common-helpers/src/run-command.ts

```diff
@@ -7,6 +7,7 @@ export function runCommand(
 ): ChildProcessPromise<PromiseResult<string>> {
   const subprocess = exec(command, {
     cwd: path,
+    maxBuffer: 1024 * 1024 * 1024,
```

Comment:
Why is this needed and what are the implications?

Reply:
Cdktf logs the progress while deploying the application. The problem is caused by `cdktf` sending a lot of data (more than 1MB) to stdout or stderr: `Error: ChildProcessError: stdout maxBuffer length exceeded`

We need to increase the std buffer size